docs/sources/reference/run.md

```diff
@@ -894,7 +895,12 @@ remote server at `192.168.0.42` on port `123`
 
     $ docker run --log-driver=syslog --log-opt syslog-address=tcp://192.168.0.42:123
 
-`syslog-tag` specifies tag for syslog messages from container.
+`syslog-facility` specifies the syslog facility. This can be an integer of 0 to 23 or any of the
+following named facilities: kern, user, mail, daemon, auth, syslog, lpr, news, uucp, cron,
+authpriv, ftp, local0, local1, local2, local3, local4, local5, local6, local7
```

Comment:
(sorry for the noise; edited twice, because I wasn't sure if line 887 was an *example* or the default value)